server/auctionapp/src/main/java/com/atlantbh/auctionapp/model/enums/SortBy.java

```diff
@@ -1,7 +1,8 @@
 package com.atlantbh.auctionapp.model.enums;
 
 public enum SortBy {
-    START_DATE("startDate");
+    START_DATE("startDate"),
+    DESCENDING("descending");
```

Comment:
Why do we have descending inside SortBy enum? Rather parse descending string into `org.springframework.data.domain.Sort.Direction`
And what about other options beside startDate (name, startPrice)?